Formula/fceux.rb

```diff
@@ -1,9 +1,11 @@
 class Fceux < Formula
   desc "All-in-one NES/Famicom Emulator"
   homepage "https://fceux.com/"
-  url "https://downloads.sourceforge.net/project/fceultra/Source%20Code/2.2.3%20src/fceux-2.2.3.src.tar.gz"
-  sha256 "4be6dda9a347f941809a3c4a90d21815b502384adfdd596adaa7b2daf088823e"
-  revision 4
+  url "https://github.com/TASVideos/fceux.git",
+      tag: "fceux-2.3.0",
+      revision: "65c5b0d2a1c08db75bb41340bfa5534578926944"
```

Comment:
```suggestion
      tag:      "fceux-2.3.0",
      revision: "65c5b0d2a1c08db75bb41340bfa5534578926944"
```
Sorry, forgot these had to be aligned.